src/foam/nanos/notification/email/EmailTemplateUtil.js

```diff
@@ -0,0 +1,396 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.notification.email',
+  name: 'EmailTemplateUtil',
```

Comment:
"Util" is vague. How about a more descriptive name like "Engine" (I don't know why, but template systems are often called engines) or Service or Compiler. Is there anything email specific about this?